src/launchWizard/launchWizard.js

```diff
@@ -64,7 +64,7 @@ function getConfigValues() {
     'openInfosetDiffView'
   ).checked
   const openInfosetView = document.getElementById('openInfosetView').checked
-  const program = document.getElementById('program').value
+  const schema = document.getElementById('schema').value
```

Comment:
Hmmm. Can you help me understand why we have both a launchWizard.js and a launchWizard.ts file? I thought ts was the VSCode typed flavor of javascript, and that its use subsumed javascript inside VSCode. 

My understanding is obviously superficial. Can you help me understand why we need both in this situation?

Reply:
@mbeckerle Yes, so the reason for the TS and JS files is because the JS file directly interacts with the DOM/UI of the WebView. For the scripting of WebViews it only directly support JS and not TS, so we use JS to interact with UI and then forward that information to the TS. We also use JS to update the UI based on information sent back to the JS from the TS. So, overall JS is used mostly for DOM and UI interaction and the TS does everything else but they communicate back and forth with each other using the VSCode API. The JS version of that API in this case has a lot less capabilities than the VSCode API in the TS and that is why we don't use only JS for this as well.

Does that help or make it more confusing?

Reply:
I also I believe the reason it touched so much is because we based this on [vscode-mock-debug](https://github.com/microsoft/vscode-mock-debug). We mostly followed what they did which I believe starts with a `program` setting. So I believe that was where most of the references came from.